test/parallel/test-buffer-alloc.js

```diff
@@ -768,14 +768,8 @@ Buffer.from(Buffer.allocUnsafe(0), 0, 0);
 }
 
 // issue GH-4331
-assert.throws(() => Buffer.allocUnsafe(0xFFFFFFFF),
- common.engineSpecificMessage({
-   v8: RangeError,
-   chakracore: TypeError}));
-assert.throws(() => Buffer.allocUnsafe(0xFFFFFFFFF),
- common.engineSpecificMessage({
-   v8: RangeError,
-   chakracore: TypeError}));
+assert.throws(() => Buffer.allocUnsafe(0xFFFFFFFF));
```

Comment:
@kunalspathak Thanks for looking into this. So ChakraCore throws same error as V8.